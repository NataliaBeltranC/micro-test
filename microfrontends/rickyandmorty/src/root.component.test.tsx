import { render } from "@testing-library/react";
import Root from "./root.component";

describe("Root component", () => {
  it("should be in the document", () => {
    const { getByText, container } = render(<Root name="Testapp" />);
    expect(getByText(/Rick and Morty/i)).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
