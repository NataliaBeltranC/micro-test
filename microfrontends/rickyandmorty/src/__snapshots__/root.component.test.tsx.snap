// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root component should be in the document 1`] = `
<div>
  <button
    class="button-general"
  >
    Ver personajes de Rick and Morty
  </button>
</div>
`;
